priv/catalog/1.3.1-0B6DB2.yaml

```diff
@@ -0,0 +1,58 @@
+id: 0B6DB2
+name: SBD Pacemaker
+# name: 1.3.1
+group: SBD
+description: |
+  `SBD_PACEMAKER` value is correctly set in SBD configuration
+
+  Not applicable for GCP, AWS
+remediation: |
+  ## Abstract
+  For proper SBD fencing, make sure that the integration with Pacemaker is enabled.
+  **IMPORTANT**: Always verify these steps in a testing environment before doing so in production ones!
+
+  The SBD is not used in GCP or AWS environments.
+  ## Remediation
+  Run the following commands in order:
+
+  1. Put cluster into maintenance mode:
+     ```crm configure property maintenance-mode=true```
+  2. Stop the cluster:
+     ```crm cluster stop```
+  3. Set the SBD_PACEMAKER parameter to `yes` on `/etc/sysconfig/sbd`:
+     ```
+     [...]
+     SBD_PACEMAKER="yes"
+     [...]
+     ```
+  4. Restart the cluster:
+     ```crm cluster start```
+  5. Put cluster out of maintenance mode
+     ```crm configure property maintenance-mode=false```
+
+  ## References
+  Azure:
+
+    - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker#set-up-the-iscsi-target-server-sbd-device
+
+  AWS:
+
+
+  GCP:
+
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-1.11.5.5.4.4
+
+run_when: env.provider != "aws" && env.provider != "gcp"
```

Comment:
Is 'run_when:' already implemented and usable or is this WIP?

Reply:
@angelabriel please ignore this PR as it was discovery work and cannot be taken as reference right now

Reply:
ok, many thanks for the clarification